include/elf/riscv.h

```diff
@@ -87,6 +87,8 @@ START_RELOC_NUMBERS (elf_riscv_reloc_type)
   RELOC_NUMBER (R_RISCV_SET8, 54)
   RELOC_NUMBER (R_RISCV_SET16, 55)
   RELOC_NUMBER (R_RISCV_SET32, 56)
+  RELOC_NUMBER (R_RISCV_DELETE, 57)
```

Comment:
We didn't put R_RISCV_DELETE here because it is purely internal to BFD, so shouldn't be exposed in the ELF ABI.